sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/AspectRelationType.java

```diff
@@ -4,31 +4,44 @@
 
 package com.azure.ai.textanalytics.implementation.models;
 
-import com.azure.core.util.ExpandableStringEnum;
 import com.fasterxml.jackson.annotation.JsonCreator;
-import java.util.Collection;
+import com.fasterxml.jackson.annotation.JsonValue;
 
 /** Defines values for AspectRelationType. */
-public final class AspectRelationType extends ExpandableStringEnum<AspectRelationType> {
-    /** Static value opinion for AspectRelationType. */
-    public static final AspectRelationType OPINION = fromString("opinion");
+public enum AspectRelationType {
```

Comment:
Is this a public enum the user can see?

Reply:
There is no public enum that user can use for now. I can add it later in future PR if we want to support it.